addons/hr/views/hr_employee_views.xml

```diff
@@ -187,7 +187,7 @@
                                 <group>
```

Comment:
> commit
It's best to avoid spaces around `task - [id]` to avoid ids being interpreted as commit hashes